### ISSUE TITLE
Very sensitive scrolling on mobile

### ISSUE THREAD

**Issue by beeva-fernandobordallo** [alvarotrigo/pagePiling.js#93]
Hey there!

I'm trying to lower the sensitivity on Mobile with no result. When a user merely touches the screen in a vertical direction (swipes 1 millimetre) the page changes. I've tried setting different values for 'touchSensitivity' with no change whatsoever. Given its a percentage of the screen's height/width I've tried these values: 5 (default), 25, 50, 75, 100. No change between them. 
¿?

**Comment by Eugene1984** [comment]
Same issue here. Awesome plugin!

**Comment by ta-inventions** [comment]
Hey managed to debug and find a fix for this - 
TLDR - the fix is to set height: 100% on mobile to the element on which you applied pagepiling to,
and set touchSensitivity to something bigger than 5 - the higher the less the touch sensitivity will be.
@alvarotrigo - might worth while to mention it in the README 😁
The problem occurs since the touchSensitivity option is being used
to calculate the percentage of the height of the whole container on which you applied pagepiling to, for example:
```
$('#pagepiling').pagepiling({
        scrollingSpeed: 1000,
        touchSensitivity: 8,
});
```

will trigger the animation to the another section only if the end touch by the user is far away by 8% of the height of #pagepiling, as you can see here:
```
if (Math.abs(touchStartY - touchEndY) > (container.height() / 100 * options.touchSensitivity)) {
                            if (touchStartY > touchEndY) {
                                scrolling('down', scrollable);
                            } else if (touchEndY > touchStartY) {
                                scrolling('up', scrollable);
                            }
                        }
```

but since we add this styles when pagepiling initialize the container:
```
$(container).css({
            'overflow' : 'hidden',
            '-ms-touch-action': 'none',  /* Touch detection for Windows 8 */
            'touch-action': 'none'       /* IE 11 on Windows Phone 8.1*/
});
```

and then we have no height set to this container and every little touch trigger swiping

**Comment by alvarotrigo** [comment]
@ta-inventions good discovery!
Feel free to provide a Pull Request with the changes, as I'm no longer maintaining it in favor of [fullPage.js](https://github.com/alvarotrigo/fullPage.js), which is a way more advanced component.

**Comment by ta-inventions** [comment]
> [@ta-inventions](https://github.com/ta-inventions) good discovery! Feel free to provide a Pull Request with the changes, as I'm no longer maintaining it in favor of [fullPage.js](https://github.com/alvarotrigo/fullPage.js), which is a way more advanced component.

Hey, I've created a PR for this! Please approve it man #242
